### ISSUE TITLE
[FEATURE] Rotate Camera 180° or Flip it Horizontally

### ISSUE THREAD

**Issue by feutl** [TheTimeWalker/wallpanel-android#30]
I am running Wallpanel on a Fire HD 8 mounted with the front-camera upwards.
Still the camera stream shows an upside-down image and I have a really hard time rotating it on HA itself. So I thought, why not fix it at the source ;)
The rotate 90 and flip vertically is already in place, the 180 and flip horizontally is missing.

THANKS

**Comment by ejpenney** [comment]
Same issue here.  I see references in documentation to a "Flip Horizontal" but that function does not exist on my installation.  I'm on a Fire HD 10 (11th gen)

**Comment by ejpenney** [comment]
![image](https://user-images.githubusercontent.com/21226768/198093431-892eaf9b-61ed-49a9-bd17-b8f124accaf1.png)

![image](https://user-images.githubusercontent.com/21226768/198093456-59083dc7-2aad-455c-9179-1069b4065330.png)


It's also worth noting that the "flip" options aren't persisting.  If I select one it immediately clears.

**Comment by oneseventhree** [comment]
Same issue with me as well on a Samsung Galaxy Tab A.

A fix would be great! :)

**Comment by ejpenney** [comment]
I tried again today and the vertical flip worked for me this time, getting a 180 deg rotation.  I think, the bug may be that the setting sometimes doesn't set or get saved?  Perhaps some error is occurring and not getting handled.

Anyways, I guess it doesn't hurt to keep trying?

**Comment by oneseventhree** [comment]
> I tried again today and the vertical flip worked for me this time, getting a 180 deg rotation. I think, the bug may be that the setting sometimes doesn't set or get saved? Perhaps some error is occurring and not getting handled.
> 
> Anyways, I guess it doesn't hurt to keep trying?

Ahh I figured it out with your tip:
TLDR; I think the trick is just select Flip vertically over and over again until its flipped.

The solution:
I went to "Rotate Camera" and it said my rotation was NONE. I then selected 'Flip Vertically" and it still said NONE. I then tried Flip Vertically again and then it said:
<img width="1615" alt="Screenshot 2022-11-30 at 5 17 35 pm" src="https://user-images.githubusercontent.com/15043282/204722088-d93be5eb-c878-4871-b62f-7995c7b960aa.png">

I knew this would only flip it vertically so went back in and selected Flip vertically again one more time so it still showed:
<img width="1615" alt="Screenshot 2022-11-30 at 5 17 35 pm" src="https://user-images.githubusercontent.com/15043282/204722283-1904a10c-027e-4c0a-9c2c-1250fe681c10.png">
and now its flipped 180 degrees!


I'm not going to touch it now its finally working properly. I think its just a bug or we need a proper 180/Flip Horizontally feature

**Comment by mannebk79** [comment]
thanks i had the same issue
